projects/exui-common-lib/src/lib/components/selected-case/selected-case.component.ts

```diff
@@ -40,7 +42,13 @@ export class SelectedCaseComponent implements OnInit {
     return user.idamId;
   }
 
+  public getFeatureToggleFlag(): Observable<boolean> {
```

Comment:
@rayliangatos  we cannot have feature toggle code in library. This code has to be moved to application. Thanks

Reply:
moved the flag to webapp now